modules/nw-ingress-controller-configuration-parameters.adoc

```diff
@@ -116,6 +116,15 @@ Ciphers and the minimum TLS version of the configured security profile are refle
 
 * `WildcardsDisallowed`: Indicates only routes with a wildcard policy of `None` are admitted by the Ingress Controller. Updating `wildcardPolicy` from `WildcardsAllowed` to `WildcardsDisallowed` causes admitted routes with a wildcard policy of `Subdomain` to stop working. These routes must be recreated to a wildcard policy of `None` to be readmitted by the Ingress Controller. `WildcardsDisallowed` is the default setting.
 
+|`IngressControllerLogging`
+|* `logging` defines parameters for what should be logged where. If this field is empty, operational logs are enabled but access logs are disabled.
+* `access` describes how the client requests should be logged. If this field is empty, access logging is disabled.
+* `ContainerLoggingDestinationParameters` describes parameters for the Container logging destination type.
+* `destination` describes a destination for log messages.
+** `type` is type of destination for logs. It must be one of the following:
+*** `container` enables the Ingress Operator to configure the sidecar container named *logs* on the Ingress Controller Pod and configures the Ingress Controller to write logs to the sidecar. The logs are then available as container logs. The expectation is that the administrator configures a custom logging solution that reads logs from this sidecar. Using container logs means that logs may be dropped if the rate of logs exceeds the container runtime's or the custom logging solution's capacity.
+*** `syslog` enables logs tp be sent to a syslog endpoint. The administrator must specify an endpoint that can receive syslog messages. The expectation is that the administrator has configured a custom syslog instance.
```

Comment:
``` `syslog` ``` → ``` `Syslog` ```.

Should the `address`, `port`, and `facility` fields for `syslog` be documented as well?

Should `spec.logging.acess.httpLogFormat` be listed here?

Reply:
It looks like the `syslog` field is missing (the line you have describes the `Syslog` value for the `type` field).  So it looks like ``` `syslog` ``` should be changed to ``` `Syslog` ``` here (as I commented above) to describe the destination type value, and ``` `syslog` ``` needs to be added to describe the `syslog` field.

Reply:
Thank you! I need a bit of clarification on this one. 

Should the `syslog` and `Container` entries be in the bullet list below `type`?
"*** `syslog` enables logs to be sent to a Syslog endpoint. The administrator must specify an endpoint that can receive Syslog messages. The expectation is that the administrator has configured a custom Syslog instance."

And then a new `Syslog` entry on the same level as `type`? And in that entry include descriptions for `address`, `port`, and `facility`?

And where would `spec.logging.acess.httpLogFormat` fit?

Reply:
> Should the `syslog` and `Container` entries be in the bullet list below `type`?

``` `Syslog` ``` and ``` `Container` ``` should be below ``` `type` ``` (see below).

> And then a new `Syslog` entry on the same level as `type`? And in that entry include descriptions for `address`, `port`, and `facility`?

The old entry should be renamed to ``` `Syslog` ```, and the new entry would be ``` `syslog` ```.  Yeah, the new entry should go on the same level as ``` `type` ``` and ``` `container` ```, and ``` `address` ```, ``` `port` ```, and ``` `facility` ``` would be described or listed under ``` `syslog` ``` (see below).

> And where would `spec.logging.acess.httpLogFormat` fit?

It would go under ``` `access` ```:

```
* `access` describes how client requests should be logged. If this field is empty, access logging is disabled.
** `destination` describes a destination for log messages.
*** `type` is the type of destination for logs:
**** `Container` specifies that logs should go to a sidecar container; the Ingress Operator configures the container, named *logs*, on the Ingress Controller Pod and configures the Ingress Controller to write logs to the container. The expectation is that the administrator configures a custom logging solution that reads logs from this container. Using container logs means that logs may be dropped if the rate of logs exceeds the container runtime capacity or the custom logging solution capacity.
**** `Syslog` specifies that logs should be sent to a syslog endpoint. The administrator must specify an endpoint that can receive syslog messages. The expectation is that the administrator has configured a custom syslog instance.
*** `container` describes parameters for the `Container` logging destination type. Currently, there are no parameters for container logging, so this field should be empty.
*** `syslog` describes parameters for the `Syslog` logging destination type:
**** `address` is the IP address of the syslog endpoint that receives log messages.
**** `port` is the UDP port number of the syslog endpoint that receives log messages.
**** `facility` specifies the syslog facility of log messages. If this field is empty, the facility is `local1`. Otherwise, it must specify a valid syslog facility: `kern`, `user`, `mail`, `daemon`, `auth`, `syslog`, `lpr`, `news`, `uucp`, `cron`, `auth2`, `ftp`, `ntp`, `audit`, `alert`, `cron2`, `local0`, `local1`, `local2`, `local3`. `local4`, `local5`, `local6`, or `local7`.
** `httpLogFormat` specifies the format of the log message for an HTTP request. If this field is empty, log messages use the implementation's default HTTP log format. For HAProxy's default HTTP log format, see link:the HAProxy documentation[http://cbonte.github.io/haproxy-dconv/2.0/configuration.html#8.2.3].
```

Reply:
Thank you! Updated.